syllabus.md

```diff
@@ -183,23 +181,22 @@ of the assignment as you can based on what we have covered in class by the
 following class period. Therefore, even if something unexpected happens at the
 last minute you should already be close to done with the assignment. This policy
 also allows rapid feedback to be provided to students by returning assignments
-quickly, which is crucial to learning.
+quickly.
 
 
 ### Assignment policy
 
 Assignments are due Monday night by 11:59 pm Eastern Time. Assignments should be
-submitted via email to [{{ page.email }}](mailto:{{ page.email }}) with the
-subject line:
- 
-```Data Carpentry Assignment X```, where `X` is the number of the assignment.
+submitted via Canvas.
 
 
 ### Course Technology
 
-Students are required to provide their laptops and to install free and open
+Students are required to provide their own laptops and to install free and open
 source software on those laptops. Support will be provided by the instructor in
-the installation of required software.
+the installation of required software. If you don't have access to a laptop
+please contact the instructor and they will do their best to provide you with
+one.
```

Comment:
Thanks. Apparently my Jekyll's gotten a little rusty :smile:. Let's add that note on the setup page and I'll let you take care of that if you have time.